src/redux/books/books.js

```diff
@@ -2,30 +2,13 @@
 const ADD = 'bookstore-1/books/ADD';
 const UPDATE = 'bookstore-1/books/UPDATE';
 const DELETE = 'bookstore-1/books/DELETE';
-const LOAD = 'bookstore-1/books/LOAD';
 
-const defaultState = [
-  {
-    id: 1,
-    author: 'Suzanne Collins',
-    title: 'The Hunger Games',
-  },
+const defaultState = [];
 
-  {
-    id: 2,
-    author: 'Leo Toystoy',
-    title: 'Anna Karenina',
-  },
-
-  {
-    id: 3,
-    author: 'Franck Herbert',
-    title: 'Dune',
-  },
-];
+const APILINK = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/01GJNSV0FWZ2CG4E22V5TJHWHE/books';
```

Comment:
- [ ] Kindly make sure that connecting to the API is working correctly. Currently, I get the following errors when I try to connect to the API:

![image](https://user-images.githubusercontent.com/62290545/203870877-9c7af014-62fb-401d-80c2-86f0ab187221.png)

- The same thing happens when I try to add a book:
![image](https://user-images.githubusercontent.com/62290545/203870908-cf01bd6f-f090-491b-b140-9765f89c0fa5.png)


- [HINT] When I tried to see what the API returns I got the following:
![image](https://user-images.githubusercontent.com/62290545/203870978-9452bfa2-8bf8-44f5-add5-370ef9df19d7.png)